examples/osis/v1beta1/pipeline.yaml

```
@@ -0,0 +1,73 @@
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: osis.aws.upbound.io/v1beta1
+kind: Pipeline
+metadata:
+  annotations:
+    meta.upbound.io/example-id: osis/v1beta1/pipeline
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    maxUnits: 1
+    minUnits: 1
+    pipelineConfigurationBody: |
+      version: "2"
+      example-pipeline:
+          source:
+              http:
+                  path: "/example"
+          sink:
+          - s3:
+              aws:
+                  sts_role_arn: "arn:aws:iam::153891904029:role/example-osi-pipeline-role"
```

We can use following as we have this value in uptest datasource.

```suggestion
                  sts_role_arn: "arn:aws:iam::${data.aws_account_id}:role/example-osi-pipeline-role"
```